### DISCUSSION TITLE
Is the Traefik env non-persistent?

### DISCUSSION THREAD

**Discussion by xinghejd** [Dokploy/dokploy#387] [Q&A] [answered]
I had already updated and locked the Traefik's env  with the DNS provider’s API key, but it was lost after I restarted the server.

**Comment by Siumauricio** [comment, marked as answer]
Hi, Looks you are right, the env disappear when you update, I will create a issue, thanks. #389

**Comment by Siumauricio** [comment]
@xinghejd I released the v0.7.2 let me know if its works!

**Reply by xinghejd** [reply]
Thanks，it works.

**Comment by Fabback** [comment]
@Siumauricio I don't know if I should open another discussion or if an issue should be created, but I am experiencing some similar behavior, it's not really consistent but sometimes hitting the "reload" action for traefik on a remote server, it looses my custom env variables on this server (and the DNS api key with it). It's been doing that for a few weeks for sure, but I don't seem to have same experience on every server (some are debian LXCs, other VMs, and ubuntu laptops)
Is there anyway I can provide more info for this to be resolved ? or is that a case of "better start over" ?
